x-pack/plugins/enterprise_search/public/applications/esre/components/esre_guide/esre_guide.tsx

```diff
@@ -7,23 +7,84 @@
 
 import React from 'react';
 
+import {
+  EuiImage,
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiHorizontalRule,
+} from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
 
+import analyticsIllustration from '../../../../assets/images/analytics.svg';
+import scalableIllustration from '../../../../assets/images/scalable.svg';
+import simplifyIllustration from '../../../../assets/images/simplify.svg';
 import { SetEsreChrome as SetPageChrome } from '../../../shared/kibana_chrome';
 import { EnterpriseSearchEsrePageTemplate } from '../layout/page_template';
 
+import { EsreDocsSection } from './esre_docs_section';
+import { MeasurePerformanceSection } from './measure_performance_section';
+import { RankAggregationSection } from './rank_aggregation_section';
+import { SemanticSearchSection } from './semantic_search_section';
+
 export const EsreGuide: React.FC = () => {
   return (
     <EnterpriseSearchEsrePageTemplate
       restrictWidth
+      bottomBorder={false}
       pageHeader={{
         pageTitle: i18n.translate('xpack.enterpriseSearch.esre.guide.pageTitle', {
           defaultMessage: 'Enhance your search with ESRE',
         }),
       }}
     >
       <SetPageChrome />
-      <p>ESRE placeholder</p>
+      <EuiPanel color="transparent" paddingSize="none">
+        <EuiFlexGroup justifyContent="spaceBetween" direction="column" responsive>
+          <EuiFlexItem grow>
+            <EuiPanel color="subdued" hasShadow={false}>
+              <EuiFlexGroup direction="row" justifyContent="spaceBetween" alignItems="center">
+                <EuiFlexItem grow={false}>
+                  <EuiImage size="m" src={simplifyIllustration} alt="Simplify" />
```

Comment:
alt messages should be translated, but if these images are purely decorative (ie there's no functional information in them) the alt should be an empty string (`alt=""`). That way screen readers know to skip them.